app/src/main/res/values/strings.xml

```diff
@@ -31,10 +31,18 @@
   <string name="concept_card_toolbar_title">Concept Card</string>
   <string name="concept_card_close_icon_description">Clicking on this will close the concept card.</string>
   <string name="revision_card_toolbar_title">Revision Card</string>
+  <string name="unsaved_exploration_dialog_title">Leave to Topic Page?</string>
+  <string name="unsaved_exploration_dialog_description">Your progress will not be saved.</string>
+  <string name="unsaved_exploration_dialog_leave_button">Leave</string>
+  <string name="unsaved_exploration_dialog_cancel_button">Cancel</string>
   <string name="stop_exploration_dialog_title">Leave to Topic Page?</string>
-  <string name="stop_exploration_dialog_description">Your progress will not be saved.</string>
+  <string name="stop_exploration_dialog_description">Your progress has been saved.</string>
   <string name="stop_exploration_dialog_cancel_button">Cancel</string>
   <string name="stop_exploration_dialog_leave_button">Leave</string>
+  <string name="progress_database_full_dialog_title">Maximum storage capacity reached</string>
+  <string name="progress_database_full_dialog_continue_button">Continue</string>
```

Comment:
This experience seems confusing to me--'Continue' doesn't indicate to the user that they are about to delete progress. What do the PRD & mocks specify in this case?

Reply:
The mocks say continue. Although they also say click to continue in the dialog description. What should the buttons be here?

![Screenshot from 2021-07-15 11-27-51](https://user-images.githubusercontent.com/53645584/125736406-10d35be4-f54e-43dc-ab6d-7ea4bd7d8418.png)

Reply:
And also if I add the line "Click continue to proceed", do I add it in continuation with the dialog description or after leaving a line like shown in the dialog description.

Reply:
I'm not sure that I understand @MaskedCarrot. What does the current UI look like?

Reply:
As discussed over chat, we resolved the confusion & the code has been cleaned up to be a lot clearer.